addons/escoria-core/game/core-scripts/esc_item.gd

```diff
@@ -468,6 +468,17 @@ func start_talking():
 			and _movable.last_dir < animations.speaks.size():
 		if get_animation_player().is_playing():
 			get_animation_player().stop()
+
+		var talk_pose_scale = -1 \
+				if animations.speaks[_movable.last_dir].mirrored else 1
+
+		# Do not flip the entire character, because that would conflict
```

Comment:
That was because I used the same block of code as in `ESCMovable`:
https://github.com/godot-escoria/escoria-demo-game/blob/7aca986cf7e5e316876f49e972859030284c64d3/addons/escoria-core/game/core-scripts/behaviors/esc_movable.gd#L341-L348

Reply:
Fixed